### PR TITLE
networking

### PR DESCRIPTION
**Pull request by yoshuawuyts**
## WIP
- [x] http
- [x] sse
- [x] websockets

### CODE REVIEW

File: content/networking.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+```js
+var choo = require('choo')
+
+var app = choo()
+app.store((state, emitter) => {
+  state.tweets = []
+  emitter.on('fetch-tweets', (username) => {   // 1.
+    window.fetch(`/${username}/tweets`)        // 2.
+      .then((res) => res.json())               // 3.
+      .then((json) => JSON.parse(json))        // 4.
```
Comment (bendik) [inline content/networking.md]: `JSON.parse()` doesn't return a promise, and fetchs `res.json()` parses the json for you while returning a promise, so just delete this line 👍

Comment (yoshuawuyts) [reply content/networking.md]: Oh, nice catch! - I haven't used the `fetch` API much haha 😅

File: content/networking.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+that includes high-level APIs to do networking, sandboxed code execution and
+disk access. It runs on almost every platform, behaves similarly everywhere, and
+is always kept backwards compatible.
+
+An important part of this machinery is networking. In the browser, there's 8 (or
+so) different ways to access the network:
+
+- By navigating to a new page
+- Through `<script>`, `<link>`, & other tags.
+- Using `prefetch` headers.
```
Comment (goto-bus-stop) [inline content/networking.md]: i think this should be `preload`—`preload` immediately starts loading the resource, `prefetch` _maybe_ loads the resource at some point in the future if the browser thinks it has some spare time. `preload` is good for resources that always have to be loaded; `prefetch` is good for resources that _may_ be needed later (like split bundles)

Comment (yoshuawuyts) [reply content/networking.md]: aye!

**Comment by yoshuawuyts** [comment]
Integrated feedback, and finished up the websockets part! :grin:

**Pull request merged**